contract/tests/test_multicompany.py

```diff
@@ -8,12 +8,13 @@ class ContractMulticompanyCase(TestContractBase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        chart_template = cls.env.ref("l10n_generic_coa.configurable_chart_template")
         cls.company_obj = cls.env["res.company"]
         cls.company_1 = cls.env.ref("base.main_company")
         vals = {"name": "Company 2"}
         cls.company_2 = cls.company_obj.create(vals)
-        chart_template.try_loading(company=cls.company_2)
+        cls.env["account.chart.template"].try_loading(
+            "generic_coa", company=cls.company_2, install_demo=False
```

Comment:
how about this solution?

Reply:
What happens if there's no `l10n_xx` installed? I repeat my initial question: Is it really needed to create a new company for the tests?

Reply:
I did not write this test. But my assumption was that you need 2 companies for a multicompany test. 
I guess in the default installation there is only one company, correct? This would be not enough for a multi company field.

Deleting the test would also prevent creating a second company, but I don't think you have this in mind. I need more than a question I cannot answer.

Reply:
Yeah, I suppose this came as a regression test for a multi-company problem in the past, but it's better to keep it. Let's continue with your last change, and see when integrating this module in future installations my CI blows away.